ProcessMaker/processmaker
# FOUR-10027 | Develop Asset Addition, Duplication, and Delete Confirmation Modals

# Issue
Ticket: [FOUR-10027](https://processmaker.atlassian.net/browse/FOUR-10027)


Develop Modals for Asset Addition, Duplication, and Deletion.
Create Ellipsis Menu with Search Bar for Add Asset Dropdown.

# How to Test
1. Go to branch `feature/FOUR-10027` in `processmaker`.
2. Go to branch `feature/FOUR-10027` in `package-projects`. Install the package.
3. Go to Designer → Projects. Have a Project created and Assets associated to the project.
4. Open a Project. Click on +ASSET.
	- There should be an Ellipsis Menu with different types of Assets to add.
	- There should be a search bar on the Ellipsis Menu.
		NOTE: Search bar functionality is not completed.
	- Clicking on an Asset in the Ellipsis Menu should display a 'Create {Asset Type}' Modal.
5. Click on the Ellipsis Menu for one of the Assets in your Project.
	- Clicking the 'Duplicate' button should display a 'Duplicate {Asset Type}: {Asset Name}' Modal.
		- The name of the Asset should prefill with 'Copy' added to the end of the name.
	- Clicking the 'Remove' button should display a 'Remove Asset' Modal.

ci:next

# Code Review Checklist


[FOUR-10027]: https://processmaker.atlassian.net/browse/FOUR-10027?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
